docs/sphinx/sst/running_simeng_with_sst.rst

```diff
@@ -0,0 +1,361 @@
+
+.. code-block:: python
+
+   l1cache = sst.Component("l1cache.mesi", "memHierarchy.Cache")
+   l1cache.addParams({
+        "access_latency_cycles" : "2",
+        "cache_frequency" : "2Ghz",
+        "replacement_policy" : "nmru",
+        "coherence_protocol" : "MESI",
+        "associativity" : "4",
+        "cache_line_size" : "64",
+        "debug" : 1,
+        "debug_level" : 1,
+        "L1" : "1",
+        "cache_size" : "200KiB",
+    })
+
+Configuring the MemoryController and MemBackend
+***********************************************
+
+As discussed above, the ``MemoryController`` and the ``MemBackend`` are the last levels in the memory hierarchy. The ``MemBackend`` is a subcomponent of the 
+``MemoryController`` (``slot: backend``) and for the SimEng simulation currently, the ``SimpleMem`` backend is used. Other backends are also supported.
+
+All configuration parameters (and their documentation), ports, and slots for ``MemoryController`` and ``MemBackend`` can be found using the commands: ``sst-info memHierarchy.MemController``  
+and ``sst-info memHierarchy.simpleMem``. The default memory controller and backend parameters provided by the installation are defined below:
+
+.. code-block:: python
+
+   # Memory controller
+    memctrl = sst.Component("memory", "memHierarchy.MemController")
+    memctrl.addParams({
+        "clock" : "1GHz",
+        "request_width" : "64",
+        "debug" : 1,
+        "debug_level" : 1,
+        "addr_range_end" : 1*1024*1024*1024-1,
+    })
+
+    # Memory model
+    memory = memctrl.setSubComponent("backend", "memHierarchy.simpleMem")
+    memory.addParams({
+        "access_time" : "1ns",
+        "mem_size" : "1GiB",
+    })
+
+Connecting Components with links
+********************************
+In the last step, we need to connect all components defined above together so that they can communicate with each other and exchange memory requests. As mentioned 
+above, the ``high_network_0`` port of the Cache needs to be connected to higher cache levels or the core. Since the SimEng core interfaces with the memory system 
+through ``StandardInterface``, the cache will be connected to it. The default core to memory hierarchy component link provided by the installation are defined 
+below:
+
+.. code-block:: python
+    
+    cpu_to_cache_link = sst.Link("link1")
+    cpu_to_cache_link.connect((iface, "port", "100ps"), (l1cache, "high_network_0", "100ps"))
+    # or
+    cpu_to_cache_link = sst.Link("link1")
+    iface.addLink(cpu_to_cache_link, "port", "100ps")
+    l1cache.addLink(cpu_to_cache_link, "high_network_0", "100ps")
+
+Now that the SimEng core has been connected to the cache, the cache needs to connect to the lower levels of memory. For this configuration, it will be connected 
+to the memory controller otherwise it will be connected to lower levels of cache i.e. L2. As mentioned earlier the cache connects to lower levels through the 
```

Comment:
*..."to the memory controller. Otherwise, it could be connected to lower levels of cache i.e. L2."